src/metabase/search/scoring.clj

```diff
@@ -120,21 +124,27 @@
   (comp (partial * factor) scorer))
 
 (def ^:private match-based-scorers
+  ;; If the below is modified, be sure to update `text-score-max`!
   [consecutivity-scorer
    total-occurrences-scorer
-   (weigh-by 1.5 exact-match-scorer)])
+   (weigh-by 2 exact-match-scorer)])
```

Comment:
It's so frustrating when you type in exactly what you want and it doesn't show up